config/locales/devise.en.yml

```diff
@@ -4,21 +4,27 @@ en:
   devise:
     confirmations:
       confirmed: "Your email address has been successfully confirmed."
+      confirmed: |
+        Thank you, your email address is now verified.
+
+        You can now finish setting up your early training account. We will ask you some questions about setting and your experience. This is so we can make sure we are providing training that is useful to everyone in the early years sector.
```

Comment:
The sentence _"We will ask you some questions about setting and your experience."_ doesn't appear to be grammatically correct. Should it be:
```
We will ask you some questions about your experience setting up your account.
```

Reply:
I think `setting` here is a noun i.e. "your setting's postcode" - it is confusing to have it also used as a verb.

Reply:
The DfE love making verbs into nouns.

Reply:
Then should it be:
```
We will ask you some questions about your setting and experience.
```